DataAccessLayer/Crud/Implementation/CategoryCrud.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DataAccessLayer.DataBaseModels;
+using DataAccessLayer.Crud.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataAccessLayer.Crud.Implementation
+{
+    public class CategoryCrud : ICrudDictionary<Category>
+    {
+        public List<Category> GetAll()
+        {
+            try
+            {
+                using (LanguageSkillsDBContext db = new LanguageSkillsDBContext())
+                {
+                    return db.Categories.ToList();
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return new List<Category>();
+            }
+        }
+            
+
+        public Category Get(int id)
+        {
+            try
+            {
+                using (LanguageSkillsDBContext db = new LanguageSkillsDBContext())
+                {
+                    return db.Categories.Find(id);
```

Comment:
Did you test this method?
"Find" method has a predicate as a parameter.
Please check and fix if required.

Another comment - you should always check all your code locally before commit.

Reply:
Also relevant to all other classes below